configs/doee-electricbike-proj.nrel-op.json

```diff
@@ -0,0 +1,107 @@
+{
+  "url_abbreviation": "doee-electricbike-proj",
+  "version": 1,
+  "ts": 1707330356754,
+  "server": {
+    "connectURL": "https://doee-electricbike-proj-openpath.nrel.gov/api/",
+    "aggregate_call_auth": "user_only"
+  },
+  "opcode": {
+    "autogen": true,
+    "subgroups": [
+      "test",
+      " default"
+    ]
+  },
+  "intro": {
+    "program_or_study": "program",
+    "start_month": "01",
+    "start_year": "2024",
+    "mode_studied": "e-bike",
+    "program_admin_contact": "For questions about the program, please contact Robera Tasissa (robera.tasissa@dc.gov) or Laura Pimentel (laura.pimentel@dc.gov).",
+    "deployment_partner_name": "Energy Administration at District of Columbia Department of Energy and Environment",
+    "translated_text": {
+      "en": {
+        "deployment_partner_name": "Energy Administration at District of Columbia Department of Energy and Environment",
+        "deployment_name": "E-bike library project",
+        "summary_line_1": "The EA at DOEE aims to incentivize commuters to adopt energy-efficient, non- polluting modes of transportation and maximize air quality benefits.",
+        "summary_line_2": "This pilot program is providing e-bikes to low-income program participants in Southwest DC.",
+        "summary_line_3": "If you are a program participant, this app can help you log your trips and compete against others to benefit our environment.",
+        "short_textual_description": "The pilot will allow District residents to try out various types of e-bikes at no-cost and encourage a mode shift that reduces emissions, increases mobility for low-income residents, and demonstrates an e-bike library concept applicable to everyday life. At the end of the pilot, the data tracked using the NREL Openpath app will be used to understand e-bike use behavior, mode-shift, and any barriers to adoption.",
+        "why_we_collect": "The data collected during the program will be used to understand: how e-bikes are used, including when and how often, distance travelled, and the preferred type of e- bikes.",
+        "research_questions": [
+          "Travel behavior using e-bikes?",
+          "Barriers to e-bike use and adoption in Southwest DC?",
+          ""
```

Comment:
Here

Reply:
I think it's OK, it could look odd with an empty string plugged into the text, the safest bet might be to just take those out so the list has a length of 2, but I'm not sure it matters. If you test in the devapp you could see how it looks?

Reply:
In testing, I realized that we are not currently even using the `research_questions` field in the app: 
<img width="250" alt="Screenshot 2024-02-07 at 3 27 53 PM" src="https://github.com/e-mission/nrel-openpath-deploy-configs/assets/61334340/15ae261c-a1de-4dfe-aece-2381763efc27">

When compared to `why_we_collect`:
<img width="250" alt="Screenshot 2024-02-07 at 3 28 02 PM" src="https://github.com/e-mission/nrel-openpath-deploy-configs/assets/61334340/0dfcf024-2320-467c-b93b-60aec65e2cc4">

Pictures of my testing

| ![Simulator Screenshot - iPhone 13 - 2024-02-07 at 15 25 00](https://github.com/e-mission/nrel-openpath-deploy-configs/assets/61334340/47e57e49-77db-4963-8fa9-7f6bae0d21c7) | ![Simulator Screenshot - iPhone 13 - 2024-02-07 at 15 24 46](https://github.com/e-mission/nrel-openpath-deploy-configs/assets/61334340/e8359324-ba99-44fc-a26b-66d9d3ff8935) |
|---|---|
| ![Simulator Screenshot - iPhone 13 - 2024-02-07 at 15 24 42](https://github.com/e-mission/nrel-openpath-deploy-configs/assets/61334340/161439a3-e4a3-410d-a21e-b70a474fcd5a) | ![Simulator Screenshot - iPhone 13 - 2024-02-07 at 15 24 07](https://github.com/e-mission/nrel-openpath-deploy-configs/assets/61334340/b31a37ab-0cf7-4407-9af9-916c7701e172) |

One thing I noticed, this section is written weirdly:
<img width="386" alt="image" src="https://github.com/e-mission/nrel-openpath-deploy-configs/assets/61334340/9970eabd-d8de-4fa0-97a1-75f84543ad86">

Reply:
Since these 2 things sort of go together (both are changes that would affect the `consent-text.html` files, should I put up a PR to fix these 2 things? @shankari how would we want to include the research questions into the consent text?

Reply:
Which are the two things?
The research questions were in the consent text earlier - you can look at the snapshots back in May to see what the consent looked like then. There is also no `consent-text.html` any more so I am not sure what this means.

Reply:
@shankari To answer your question about which are the two things:
1. ~~Put the research questions somewhere in the Privacy Policy screen on the app~~ Me and @Abby-Wheelis  met today and decided that this change is unnecessary for 2 reasons: (1) the `research_questions` field in the config, while is not being used in the app, is being used in the program join pages, and (2) the Privacy policy on the app is already long enough, adding more to it would just muddle it further
2. The textual glitch I showed in my screenshots on the Privacy policy screen above: `"Program Administrators from Energy Administration at District of Colombia Department of Energy and Environment to , and NREL OpenPATH developers for debugging"` @Abby-Wheelis 's phone on the production version of NREL-commute is not showing this, but there are emojis there, so this could be a regression from my [accessibility changes](https://github.com/e-mission/e-mission-phone/pull/1118). I am looking into why my emulator is showing this way